walkthroughs/howto-servicediscovery-cloudmap/README.md

```diff
@@ -15,13 +15,16 @@ Set or export the following environment variables with appropriate values for yo
 
 ```bash
 # Your AWS account ID
-export AWS_ACCOUNT_ID=999999999999
+export AWS_ACCOUNT_ID=<YOUR AWS ACCOUNT ID>
```

Comment:
git didn't let me commit without this change. It thought this was a secret that I am committing. I could have done `git commit --no-verify`. But anyway.